### ISSUE TITLE
Cannot import Equi7Grid after pip install v0.2.3

### ISSUE THREAD

**Issue by wpreimes** [TUW-GEO/Equi7Grid#33]
Just leaving this here in case someone else tries to install the latest version of the package.
Looks like the `data` folder is not downloaded correctly in version `0.2.3`.

I get the following error on import
```
>>> from equi7grid.equi7grid import Equi7grid
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/wpreimes/mambaforge/envs/io_utils/lib/python3.9/site-packages/equi7grid/equi7grid.py", line 74, in <module>
    class Equi7Grid(TiledProjectionSystem):
  File "/home/wpreimes/mambaforge/envs/io_utils/lib/python3.9/site-packages/equi7grid/equi7grid.py", line 95, in Equi7Grid
    _static_data = _load_static_data(__file__)
  File "/home/wpreimes/mambaforge/envs/io_utils/lib/python3.9/site-packages/equi7grid/equi7grid.py", line 69, in _load_static_data
    with open(fname, "rb") as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/wpreimes/mambaforge/envs/io_utils/lib/python3.9/site-packages/equi7grid/data/equi7grid.dat'
```

Workaround is to `pip install equi7grid==0.2.2`

However it should be easy to fix this. I assume something to do with the meta-package. Just let me know if you cannot work on this right now, I can take a look if you want.

**Comment by bbauerma** [comment]
I just add: I packaged it for v=0.2.3 using the make commands. Maybe I overlooked sth.
tagging @SwamyDev here.

**Comment by wpreimes** [comment]
Fixed in `v0.2.4`
